VoodooRMI/Utility/ButtonDevice.cpp

```diff
@@ -44,7 +45,7 @@ void ButtonDevice::stop(IOService* provider) {
 void ButtonDevice::updateButtons(int buttons) {
     uint64_t now_abs;
     clock_get_uptime(&now_abs);
-    dispatchRelativePointerEvent(0, 0, buttons, now_abs);
+    //dispatchRelativePointerEvent(0, 0, buttons, now_abs);
```

Comment:
This is why your buttons only work when sending movements. UpdateButtons is called in F03::message when it recieves the buttons. This also breaks other buttons within F30, breaking it for users which have 3 buttons below the trackpad.

Reply:
Uncommented that line. I checked if there is any other unwanted changes with git but obviously missed that. Late night coding is dangerous.